android/app/src/main/java/com/zulipmobile/MainActivity.kt

```diff
@@ -37,6 +40,11 @@ open class MainActivity : ReactActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         WebView.setWebContentsDebuggingEnabled(true)
+        // Intent is reused after quitting, skip it.
+        if ((getIntent().flags and Intent.FLAG_ACTIVITY_LAUNCHED_FROM_HISTORY) != 0) {
+            return;
+        }
+
         maybeHandleIntent(intent)
```

Comment:
nit: should have blank line before this new bit of code, and no blank line between it and the `maybeHandleIntent` below it. That helps make it visually clear that this code goes together with that line.